NOTICES.md

```diff
@@ -0,0 +1,109 @@
+Update Manager ("Provisioning Technology") for the purpose of allowing users to install software, documentation, information and/or
+other materials (collectively "Installable Software"). This capability is provided with the intent of allowing such users to
+install, extend and update Eclipse-based products. Information about packaging Installable Software is available at
+[https://www.eclipse.org/equinox/p2/repository_packaging.html](https://www.eclipse.org/equinox/p2/repository_packaging.html)
+("Specification").
+
+You may use Provisioning Technology to allow other parties to install Installable Software. You shall be responsible for enabling the
+applicable license agreements relating to the Installable Software to be presented to, and accepted by, the users of the Provisioning Technology
+in accordance with the Specification. By using Provisioning Technology in such a manner and making it available in accordance with the
+Specification, you further acknowledge your agreement to, and the acquisition of all necessary rights to permit the following:
+
+1. A series of actions may occur ("Provisioning Process") in which a user may execute the Provisioning Technology
+on a machine ("Target Machine") with the intent of installing, extending or updating the functionality of an Eclipse-based
+product.
+2. During the Provisioning Process, the Provisioning Technology may cause third party Installable Software or a portion thereof to be
+accessed and copied to the Target Machine.
+3. Pursuant to the Specification, you will provide to the user the terms and conditions that govern the use of the Installable
+Software ("Installable Software Agreement") and such Installable Software Agreement shall be accessed from the Target
+Machine in accordance with the Specification. Such Installable Software Agreement must inform the user of the terms and conditions that govern
+the Installable Software and must solicit acceptance by the end user in the manner prescribed in such Installable Software Agreement. Upon such
+indication of agreement by the user, the provisioning Technology will complete installation of the Installable Software.
+
+
+### Cryptography
+
+Content may contain encryption software. The country in which you are currently may have restrictions on the import, possession, and use, and/or re-export to
+another country, of encryption software. BEFORE using any encryption software, please check the country's laws, regulations and policies concerning the import,
+possession, or use, and re-export of encryption software, to see if this is permitted.
+
+
+## Third-party Content
+
+Checkpoint/Restore In Userspace (CRIU) (3.17.1)
+
+* License: GPL v2, LGPL v2.1 [https://github.com/checkpoint-restore/criu/blob/criu-dev/COPYING](https://github.com/checkpoint-restore/criu/blob/criu-dev/COPYING)
+* Project: https://criu.org/Main_Page
+* Source: https://github.com/ibmruntimes/criu
```

Comment:
This doesn't seem right. To my knowledge, we're not including any content from the CRIU project: we're only a client in the same sense that we're a client of Linux (on that platform), so if we don't need to mention Linux, we don't need to mention CRIU.

Reply:
OpenJ9 contains code calling CRIU APIs, see https://www.eclipse.org/projects/handbook/#ip-third-party
Items which may apply are:
- Includes of header files in C/C++
- Use of reflection or other means to reference APIs and implementation

Reply:
See also the part "We cut the dependency graph off at the platform."

Reply:
Fair enough, but then we should call out OpenSSL (and probably more).

Reply:
The OpenSSL usage is in the extensions and the notice is in https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/openj9-openjdk-notices

OMR should likely call out the CUDA Toolkit but I thought we could look at that separately, it's been missing for a long time.

Reply:
The handbook says

> We regard the platform as something that the adopter installs and agrees to the licensing terms independently from the project content. ... If a layer of software is installed as part of installing your project’s artifacts, then that layer of software is a dependency.

Users would have to install each of CRIU, CUDA and OpenSSL themselves to enable use by OpenJ9, so my interpretation is that each of those packages should be considered part of the "platform".

The JIT server feature of OpenJ9 uses OpenSSL directly, so if we decide it needs to be referenced, the mention in the extensions is not sufficient.